css/footer.css

```diff
@@ -0,0 +1,128 @@
+.l-footer{
+    flex-direction: row;
+    justify-content: center;
+    padding-top: 50rem;
+}
+
+.footercontainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+:root {  
```

Comment:
`:Root` should be at the very top of this file. In fact it should be in a global CSS file.